loops.js

```diff
@@ -51,3 +69,15 @@
 // >> 9
 // ==========================================
 
+for (i = 0; i < 10; i++) {
+
+    if (i < 3) {
+        console.log(i);
+    }
+    if ((i > 2) && (i < 6)) {
```

Comment:
de vergelijking hoeft niet ook tussen haken.

if( i = 2 && i = 6) is wat leesbaarder